rickshaw-index

```diff
@@ -339,17 +404,29 @@ sub write_queued_es_docs {
 sub wait_for_metric_descs {
     my @terms = @_;
     my $attempts = 1;
-    my $max_attempts = 20;
+    my $max_attempts = 8;
```

Comment:
Are we sure about this change?  This seems like it might be a highly variable thing based on run size, no?